src/Compiler/Interactive/fsi.fs

```diff
@@ -1406,44 +1405,29 @@ type internal FsiDynamicCompiler(
                  Some { man with  CustomAttrsStored = storeILCustomAttrs (mkILCustomAttrs codegenResults.ilAssemAttrs) }) }
 
     /// Generate one assembly using multi-assembly emit
-    let EmitInMemoryAssembly (tcConfig: TcConfig, emEnv: ILMultiInMemoryAssemblyEmitEnv, ilxMainModule: ILModuleDef, m) =
+    let EmitInMemoryAssembly (tcConfig: TcConfig, emEnv: ILMultiInMemoryAssemblyEmitEnv, ilxMainModule: ILModuleDef) =
         
         // The name of the assembly is "FSI-ASSEMBLY1" etc
```

Comment:
The comment is no longer correct, right?

In the new style, the naming is agnostics of the number (therefore only 1 IVT attribute needed), and the number just makes it to the version.

Reply:
fixed, thanks